templates/resources/metal_connection.md.tmpl

```diff
@@ -46,7 +44,7 @@ The following arguments are supported:
 * `metro` - (Optional) Metro where the connection will be created.
 * `facility` - (**Deprecated**) Facility where the connection will be created. Use metro instead; read the [facility to metro migration guide](https://registry.terraform.io/providers/equinix/equinix/latest/docs/guides/migration_guide_facilities_to_metros_devices)
 * `redundancy` - (Required) Connection redundancy - redundant or primary.
-* `type` - (Required) Connection type - dedicated or shared.
+* `type` - (Required) Connection type - dedicated, shared or shared_port_vlan.
```

Comment:
While we're in here, it would be good to replace this whole `Argument Reference` section with `{{ .SchemaMarkdown | trimspace }}`.  That might require adding/updating property descriptions in the resource schema.